src/routes/detail/+layout.svelte

```diff
@@ -3,6 +3,6 @@
 	let { children }: { children: Snippet } = $props();
 </script>
 
-<div class="bg-surface-400 h-full w-full">
+<div class="bg-surface-900 text-surface-50 h-full w-full">
```

Comment:
what has the exact same effect but might be a tiny bit smarter: 
```suggestion
<div class="dark h-full w-full">
```